src/qililab/qprogram/qblox_compiler.py

```diff
@@ -435,7 +448,8 @@ def _handle_measure(self, element: Measure):
         Args:
             element (Measure): measure operation
         """
-        play = Play(bus=element.bus, waveform=element.waveform)
+        time_of_flight = self._buses[element.bus].time_of_flight
+        play = Play(bus=element.bus, waveform=element.waveform, wait_time=time_of_flight)
```

Comment:
Do we have a default time of flight? In Q1ASM this parameter in the Play command needs to be filled, what happens if we do not add a time of flight mannually, I believe that the minimum is 4ns, should we set this as a default?

Reply:
Yes we have a default value. It's the `minimum_wait_duration` which is 4.

```
# Time of flight. Defaults to minimum_wait_duration and is updated if times_of_flight parameter is provided during compilation.
        self.time_of_flight = QbloxCompiler.minimum_wait_duration
```